nix/services.nix

```diff
@@ -0,0 +1,86 @@
+{ pkgs, lib ? pkgs.lib, services-flake, only-databases, poetry-env }: {
+  imports = [
+    services-flake.processComposeModules.default
+  ];
+
+  httpServer = {
+    enable = true;
+    uds = "process-compose.socket";
+  };
+
+  services = {
+    redis."r1" = {
+      enable = true;
+      port = 0; # disable listening via TCP
+      extraConfig = ''
+        locale-collate "C"
+        unixsocket ../redis.socket
+        unixsocketperm 777
+      '';
+    };
+    postgres."pg1" = {
+      enable = true;
+      superuser = "postgres";
+      listen_addresses = ""; # disable listening via TCP
+      socketDir = "data";
+      createDatabase = false;
+      initialScript.before = ''
+        DROP USER IF EXISTS evap;
+        DROP DATABASE IF EXISTS evap;
+        CREATE USER evap PASSWORD 'evap' CREATEDB;
+        CREATE DATABASE evap OWNER evap;
+      '';
+    };
+  };
+
+  # See https://github.com/juspay/services-flake/issues/352
+  settings.processes."r1".readiness_probe.exec.command = lib.mkForce "true";
+
+  settings.processes."npm-ci" = {
+    command = pkgs.writeShellApplication {
+      name = "npm-ci";
+      runtimeInputs = [ pkgs.nodejs ];
+      text = ''
+        set -e
+        CUR_HASH=$(nix-hash --flat <(cat ./package.json ./package-lock.json))
+        echo "Hash is $CUR_HASH"
+        if [[ -f node_modules/evap-hash && "$CUR_HASH" == "$(cat node_modules/evap-hash)" ]]; then
+            echo "Equal hash found, exiting"
+            exit 0
+        fi
+        set -x
+        npm ci
+        echo "$CUR_HASH" > node_modules/evap-hash
+      '';
+    };
+    disabled = only-databases;
+  };
+
+  settings.processes."init-django" = {
+    command = pkgs.writeShellApplication {
+      name = "init-django";
+      runtimeInputs = [ poetry-env pkgs.git pkgs.gnused ];
+      text = ''
+        set -e
+        if [[ -f evap/localsettings.py ]]; then
+            echo "Found evap/localsettings.py, exiting"
+            exit 0
+        fi
```

Comment:
When there are localsettings from a setup with vagrant, it is required to delete the entire file (or do this manually), right? I think we should include some comment about this in the tricks, as everyone with on old setup needs this. Alternatively, we can just recommend to start from a new clone right away

Reply:
To migrate from vagrant, people should run `clean-setup` once, this will delete localsettings

Reply:
should we add a comment to `nix/ticks.md`?

Reply:
the steps to migrate would then be
- `clean-setup` in a `nix develop` shell
- `nix run .#services` (-- --detached)
- `nix develop`
correct?

Reply:
After clean-setup, the normal instructions apply. I think that if we want to add a note about migrating, it should be Readme. However, I also feel that it will suffice to tell people in person